src/constants.js

```diff
@@ -198,3 +198,24 @@ export const METADATA_CONCURRENT_DOWNLOAD = 10;
  * Ledger token version to use when sending a token to ledger
  */
 export const LEDGER_TOKEN_VERSION = 1;
+
+/**
+ * How many custom tokens a tx can have when signing with Ledger
+ */
+export const LEDGER_TX_CUSTOM_TOKEN_LIMIT = 10;
+
+/**
+ * Min Ledger app version supported by this wallet
+ */
+export const LEDGER_MIN_VERSION = '1.0.0';
+
+/**
+ * Max Ledger app version supported by this wallet
+ * Obs: max not inclusive, so we do NOT support this version
+ */
+export const LEDGER_MAX_VERSION = '1.2.0';
```

Comment:
With this you are saying that you support v1.1.1, which is false as well, right? Why are you using this? And why making this not inclusive?

Reply:
The idea is that any update that would break backwards compatibility would at least bump the version to 1.2.0 (since we are at 1.1.0). So I could assume we also support 1.1.1 and not have the wallet reject future Ledger apps (if they're compatible).

Reply:
I'm thinking about this. My main concern is that we might add another feature, bump to v1.2.0 and this new feature does not break anything in this wallet version. But we would require the other users to updated the wallet.

Maybe this is not a bad thing to be honest.

Reply:
Maybe we need a third kind of version, which would be the protocol version (discussed in other moments).

We would have to check if the protocol version used by the wallet and the ledger app match, instead of checking the Ledger App version.

Both the Wallet and Ledger App versions could be bumped without changing the Protocol version, if they don't break compatibility, which would make new versions still work together.